app/src/main/res/layout/activity_login.xml

```diff
@@ -17,84 +17,86 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         android:src="@drawable/ic_logo_login"
-        android:layout_marginTop="10dp"
+        android:layout_marginTop="@dimen/main_activity_margin_top"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks